opencv/build.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+mkdir build
+cd build
+
+
+if [ "$(expr substr $(uname -s) 1 5)" == "Linux" ]; then
+  DYNAMIC_EXT="so"
+  TBB=""
+  OPENMP="-DWITH_OPENMP=1"
+  IS_OSX=0
+fi
+if [ "$(uname -s)" == "Darwin" ]; then
+  IS_OSX=1
+  DYNAMIC_EXT="dylib"
+  OPENMP=""
+  TBB="-DWITH_TBB=1 -DTBB_LIB_DIR=$PREFIX/lib -DTBB_INCLUDE_DIRS=$PREFIX/include -DTBB_STDDEF_PATH=$PREFIX/include/tbb/tbb_stddef.h"
+fi
+
+cmake ..                                                                 \
+    $TBB                                                                 \
+    $OPENMP                                                              \
+    -DCMAKE_SKIP_RPATH=1                                                 \
+    -DWITH_EIGEN=1                                                       \
+    -DBUILD_opencv_apps=0                                                \
+    -DBUILD_TESTS=0                                                      \
+    -DBUILD_DOCS=0                                                       \
+    -DBUILD_PERF_TESTS=0                                                 \
+    -DBUILD_ZLIB=1                                                       \
+    -DBUILD_TIFF=1                                                       \
+    -DBUILD_PNG=1                                                        \
+    -DBUILD_OPENEXR=1                                                    \
+    -DBUILD_JASPER=1                                                     \
+    -DBUILD_JPEG=1                                                       \
```

Comment:
I see some of these libraries here also ship with `conda` (e.g `libpng`, `zlib`, `libtiff`, `libjpeg` and others). We should definitely include these as build and runtime requirements. For any of these that are not here (e.g. JasPer), we might want to disable for now, but add a note that we need to add a recipe for them.